README.md

```diff
@@ -1 +1,94 @@
-# Online-Restaurant-Team12
\ No newline at end of file
+## You can check the app [here](https://gsg-cf05.github.io/Online-Restaurant-Team12/)
+------------------------------------------------
+## Name of Project: 💡***Tasty***
+-----------------------------------------------
```

Comment:
To make it look better and more organized:
- the title is the first heading 
- add the title without `name of the project`
::: 
# Tasty

- add the GitHub pages link [live link] with the other links or directly with the title 
:::

# [Tasty](https://gsg-cf05.github.io/Online-Restaurant-Team12/)